share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -531,6 +531,7 @@ sxc,sexcoin,sex coin,
 sha,shacoin,sha coin,
 shade,shadecoin,shade coin,
 sdc,shadowcoin,shadow coin,
+sdc, shadowcash, shadow cash
```

Comment:
@kewde You can remove this line 534 as triggers are already on line 533 `sdc,shadowcoin,shadow coin,`